semantic/src/site/elements/label.overrides

```diff
@@ -22,7 +27,26 @@
   background-color: transparent;
   color: @darkGrey;
   font-weight: @regularWeight;
-  font-size: @fontXXsmall;
   height: 20px;
   padding-left: 0;
 }
+
+.ui.label.text-italic {
```

Comment:
I don't think this needs its own class. To put a label in italics, just wrap it the text in `<em></em>` tags.

Reply:
Cool, didn't know about this one. Changing this now

Reply:
Done: 985669f